src/bcf/record.rs

```diff
@@ -131,6 +131,17 @@ impl Record {
         self.inner_mut().pos = pos;
     }
 
+    /// Return whether the filter with the given ID is set.
+    pub fn has_filter(&self, id: &[u8]) -> bool {
```

Comment:
Shouldn't this be `&[bcf::header::ID]`?

Reply:
Actually, the C API exposes `char const *` here. What do we want?